packages/components/pill/README.mdx

```diff
@@ -8,58 +8,41 @@ storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-pill--b
 typescript: ./src/Pill.tsx
 ---
 
-import { InfoCircleIcon } from '@contentful/f36-icons';
-
 Pills represent a set of one or multiple objects which can be entered or changed by the user. By default, pills can be removed by the user. When necessary, the order of pills can be changed by the user using drag and drop. They are used to represent complex entities in small blocks such as contact details. Contentful Pills are are used to render transition locales and tag elements.
```

Comment:
```suggestion
Pills represent a set of one or multiple objects which can be entered or changed by the user. By default, pills can be removed by the user. When necessary, the order of pills can be changed by the user using drag and drop. They are used to represent complex entities in small blocks such as contact details. Contentful Pills are used to render transition locales and tag elements.
```

Could you rephrase this description a bit? For example, first three sentences don't really make sense to me